src/instance_check.jl

```diff
@@ -4,29 +4,31 @@
 const HAS_OPENING_TIME_WINDOWS = 1
 const HAS_MULTIPLE_OPENING_TIME_WINDOWS = 2
 const HAS_FLEXIBLE_OPENING_TIME_WINDOWS = 3
+const HAS_A_RECHARGING_STATION = 4
 
 # Product based features #
-const HAS_PRODUCT_CONFLICT_CLASSES = 4
-const HAS_PRODUCT_PROHIBITED_PREDECESSOR_CLASSES = 5
-const HAS_PRODUCT_PICKUPONLY_SHARING_CLASSES = 6
-const HAS_PRODUCT_DELIVERYONLY_SHARING_CLASSES = 7
-const HAS_PRODUCT_SHIPMENT_SHARING_CLASSES = 8
+const HAS_PRODUCT_COMPATIBILITY_CONFLICT_ISSUES = 10
```

Comment:
I prefer ISSUES is vague and is mainly a "synonym" to feature. One could add it to most of the names. So I prefer to just use

HAS_PRODUCT_COMPATIBILITY_CONFLICTS
HAS_PRODUCT_PROHIBITED_PREDECESSORS